helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -829,7 +829,7 @@ image:
     # The registry and name of the opentelemetry collector image to pull
     repository: quay.io/signalfx/splunk-otel-collector
     # The tag of the Splunk OTel Collector image, default value is the chart appVersion
-    tag: ""
+    tag: "0.59.1"
```

Comment:
This shouldn't bed needed after addressing [rmfitzpatrick](https://github.com/rmfitzpatrick)'s comment.

Reply:
I'll revert right now.